Slider/Assets/Scripts/Utility/TileUtil.cs

```diff
@@ -9,4 +9,13 @@ public static Vector2Int WorldToTileCoords(Vector2 posInWorld)
         //L: Tiles are positioned based on the bottom left corner, so need to add small amount to avoid off by 1 error.
         return new Vector2Int(Mathf.RoundToInt(posInWorld.x+epsilon), Mathf.RoundToInt(posInWorld.y+epsilon));
     }
+
+    /// <summary>
+    /// Attach the passed in GameObject as a child of the transform of the STile it is underneath.
+    /// </summary>
+    public static void AttachToTileUnderneath(GameObject objectToAttach)
+    {
+        Transform tileUnderneath = SGrid.GetSTileUnderneath(objectToAttach).transform;
```

Comment:
might want a null check for ` SGrid.GetSTileUnderneath(objectToAttach)`

Reply:
Its busted af anyway so i dont care :D